packages/@react-spectrum/button/test/Button.test.js

```diff
@@ -273,6 +273,40 @@ describe('Button', function () {
     expect(eventDown.defaultPrevented).toBeFalsy();
   });
 
+  // is pending state
+  it('displays a spinner after a short delay when isPending prop is true', function () {
+    jest.useFakeTimers();
+    let onPressSpy = jest.fn();
+    function TestComponent() {
+      let [pending, setPending] = useState(false);
+      return (
+        <Button onPress={(pending) => { setPending(true); onPressSpy(); }} isPending={pending}>
+          Click Me
+        </Button>
+      );
+    }
+    let {getByRole} = render(<TestComponent />);
+    let button = getByRole('button');
+    expect(button).toHaveAttribute('aria-disabled', 'false');
+    fireEvent.click(button);
+    expect(button).toHaveAttribute('aria-disabled', 'true');
+    // Multiple clicks shouldn't call onPressSpy
+    fireEvent.click(button);
```

Comment:
use `triggerPress` instead of `fireEvent.click`, it's more accurate to the events that get fired in a real browser